opennms-base-assembly/src/main/filtered/etc/snmp-graph.properties.d/opennms-graph.properties

```diff
@@ -38,7 +38,10 @@ OpenNMS.Newts.Index.Inserts, \
 OpenNMS.Newts.Index.Update.Latency, \
 OpenNMS.Newts.Index.Delete.Latency, \
 OpenNMS.Cassandra.Cluster1.Requests, \
-OpenNMS.Cassandra.Cluster1.Request.Latency
+OpenNMS.Cassandra.Cluster1.Request.Latency, \
+org.opennms.core.ipc.sink.kafka.OpenNMS.Sink.Heartbeat.lag.HeartbeatLag.AttributeReport, \
```

Comment:
Can we rename this to `OpenNMS.Sink.Heartbeat.Lag` instead of  `org.opennms.core.ipc.sink.kafka.OpenNMS.Sink.Heartbeat.lag.HeartbeatLag.AttributeReport` so that it fits in with the other report names? (And similarly for the Trap one bellow)